gf-webapp-admin/src/appContext.ts

```diff
@@ -3,10 +3,10 @@ import firebase from "firebase/compat/app";
 import {Auth} from "firebase/auth";
 import {Firestore} from "firebase/firestore";
 
-interface IAppContext {
+interface AppContext {
   firebase: typeof firebase,
   auth: Auth,
   firestore: Firestore
 }
 
-export const AppContext = createContext<IAppContext>({} as IAppContext);
\ No newline at end of file
+export const AppContext = createContext<AppContext>({} as AppContext);
```

Comment:
Is it  possible to avoid construct as...?

Reply:
There are contradictions in the types that I cannot solve. So I leave it as it is.